examples/src/main/java/io/grpc/examples/routeguide/RouteGuideClient.java

```diff
@@ -271,10 +269,40 @@ public static void main(String[] args) throws InterruptedException {
     }
   }
 
-  private static void info(String msg, Object... params) {
+  /**
+   * Make it visible for unit test, so that test can verify state changes by spying on it.
+   */
+  @VisibleForTesting
+  void info(String msg, Object... params) {
     logger.log(Level.INFO, msg, params);
   }
 
+  /**
+   * Make it visible for unit test, so that test can verify state changes by spying on it.
+   */
+  @VisibleForTesting
+  void warning(String msg, Object... params) {
```

Comment:
Why does the logger need to be spied?  It's intended for humans, not program consumption.